app/css/ui-kit/colors.css

```diff
@@ -338,3 +338,10 @@
 
     @apply text-textColor5 bg-backgroundColorB;
 }
+
+@media (prefers-color-scheme: dark) {
+    body.theme-unset {
+        background: red;
+        @reuse .theme-dark;
+    }
+}
```

Comment:
If, in the bit above this where you set `.theme-dark`, you could instead do:
```.theme-dark, @media (prefers-color-scheme: dark) .theme-unset {
...
}
```

then that would be better. But I don't think you can use the `,` and media queries.

If `@reuse` works then this is a fine alternative though.